### PR TITLE
fix(migration): Add missing migration attributes

### PR DESCRIPTION
**Pull request by nickvergessen**
Was missing from community PR #47253

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by come-nc** [comment]
ref: https://github.com/nextcloud/server/pull/46476
@nickvergessen Why only this migrations? None of the core migrations have the attributes it seems.

**Comment by nickvergessen** [comment]
> None of the core migrations have the attributes it seems.

30+ migrations are supposed to have it.

File: core/Migrations/Version30000Date20240815080800.php [hunk 1]
```diff
@@ -12,9 +12,12 @@
 use Closure;
 use OCP\DB\ISchemaWrapper;
 use OCP\DB\Types;
+use OCP\Migration\Attributes\AddColumn;
+use OCP\Migration\Attributes\ColumnType;
 use OCP\Migration\IOutput;
 use OCP\Migration\SimpleMigrationStep;
 
+#[AddColumn(table: 'webauthn', name: 'user_verification', type: ColumnType::STRING)]
```
Comment (AndyScherzinger) [inline core/Migrations/Version30000Date20240815080800.php:20]: @nickvergessen String or Boolean? (I guess the later depends on the DBMS?)

Comment (nickvergessen) [reply core/Migrations/Version30000Date20240815080800.php:20]: Wrong copy paste. boolean here

**Review by AndyScherzinger — APPROVED** [review]
LGTM

**Comment by AndyScherzinger** [comment]
@come-nc they actually should have them. At least @ArtificialOwl did add them to v30 some time ago. But if there are still some missing, we should have them added 👍

**Review by SystemKeeper — APPROVED** [review]

**Pull request merged**